src/auth/Auth.h

```diff
@@ -30,6 +30,15 @@ enum {
   AUTH_MODE_MON_MAX = 19,
 };
 
+typedef enum CONN_TYPE {
```

Comment:
why not just:
class enum ConnectionType {...}; 
?

Reply:
Consistency with the things related to the connection _mode_ (secure vs crc). Plain`enums` are used there and I'm not sure whether `enum class` brings so much of value to justify refactoring.

Reply:
There is some value, I think, in not having an enum name that will never be used, and then a typedef.
I would vote for using new style for the added code, even if not refactoring the existing.

Reply:
> There is some value, I think, in not having an enum name that will never be used,

Ah, that's a good point. I missed the fact I left the name for `enum`. In the cephx implementation we do simply:

```cpp
  // -- incremental updates --
  typedef enum {
    AUTH_INC_NOP,
    AUTH_INC_ADD,
    AUTH_INC_DEL,
    AUTH_INC_SET_ROTATING,
  } IncrementalOp;
```

Will drop that.